app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1327,45 +1324,6 @@ private void clearAnswer(QuestionWidget qw) {
         qw.clearAnswer();
     }
 
-
-    /*
-     * (non-Javadoc)
-     * @see android.app.Activity#onCreateContextMenu(android.view.ContextMenu, android.view.View, android.view.ContextMenu.ContextMenuInfo)
-     */
-    @Override
-    public void onCreateContextMenu(ContextMenu menu, View v, ContextMenuInfo menuInfo) {
-        super.onCreateContextMenu(menu, v, menuInfo);
-            menu.add(0, v.getId(), 0, StringUtils.getStringSpannableRobust(this, R.string.clear_answer));
```

Comment:
Hm, that's a tough question, generally. I'd say let's remove it. If we want
them in the future it will likely be to add this functionality back so we
can pull them out of version control.


On Fri, Apr 17, 2015 at 2:06 PM, Phillip Mates <notifications@github.com>
wrote:

> In app/src/org/odk/collect/android/activities/FormEntryActivity.java
> <https://github.com/dimagi/commcare-odk/pull/231#discussion_r28616466>:
>
> > @@ -1327,45 +1324,6 @@ private void clearAnswer(QuestionWidget qw) {
> >          qw.clearAnswer();
> >      }
> >
> > -
> > -    /*
> > -     * (non-Javadoc)
> > -     * @see android.app.Activity#onCreateContextMenu(android.view.ContextMenu, android.view.View, android.view.ContextMenu.ContextMenuInfo)
> > -     */
> > -    @Override
> > -    public void onCreateContextMenu(ContextMenu menu, View v, ContextMenuInfo menuInfo) {
> > -        super.onCreateContextMenu(menu, v, menuInfo);
> > -            menu.add(0, v.getId(), 0, StringUtils.getStringSpannableRobust(this, R.string.clear_answer));
>
> Leaving in R.string.clear_answer text entry, and all the other
> corresponding text entries.
> They will be rendered unused, but might be useful in the future? Or should
> I just delete them?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-odk/pull/231/files#r28616466>.
>